partcad/src/partcad/user_config.py

```
@@ -45,6 +45,11 @@ def __init__(self):
 
         self.set_default("sentry.shutdown_timeout", 5)
         self.set_default("sentry.traces_sample_rate", 1.0)
+        self.set_default("sentry.dsn", "https://sentry.partcad.org")
+        self.set_default("sentry.environment", "production")
+        self.set_default("sentry.debug", False)
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)